packages/storybook8/stories/Concepts/TogetherMode/Docs.mdx

```diff
@@ -0,0 +1,62 @@
+import { Meta, Source } from '@storybook/addon-docs';
+
+<Meta title="Concepts/TogetherMode" />
+
+import PPTLiveSnippetText from '!!raw-loader!./snippets/TogetherMode.snippet.tsx';
+
+# TogetherMode
+
+Together Mode in Teams uses AI to place everyone in a shared virtual background, creating the feeling of being in the same room.
+It’s built to make meetings more engaging and foster stronger connections.
+
+It’s especially effective for training sessions, workshops, or team-building activities.
+A classroom setting, like the one shown below, promotes learning and collaboration, helping participants stay engaged and comfortable in a familiar atmosphere.
+
+<img style={{ width: 'auto', height: 'auto' }} src="images/together-mode/togethermode-scenario_example.png" />
+
+## Supported Scenario
+
+Together Mode is available only in Teams meetings and group calls. It can be started exclusively by Teams users with the roles of Organizer, Co-organizer, or Presenter.
+ACS (Azure Communication Services) users cannot initiate Together Mode themselves, but they can view the Together Mode stream if it’s started by a Teams user.
+
+<sub>
+  <sup>Layout Menu Displaying the Together Mode Option as Disabled When a Teams User Hasn't Started Together Mode</sup>
+</sub>
+<img style={{ width: 'auto', height: 'auto' }} src="images/together-mode/togethermode-not-active-for-acs-user.png" />
+The option to start or switch to Together Mode is disabled because ACS users do not have permission to initiate Together
+Mode.
+
+<sub>
+  <sup>Layout Menu Showing the Together Mode Option Enabled When a Teams User Has Started Together Mode</sup>
+</sub>
+<img style={{ width: 'auto', height: 'auto' }} src="images/together-mode/togethermode-active-for-acs-user.png" />
+When a Teams user starts Together Mode, the view option becomes enabled, allowing ACS users to switch to the Together
+Mode view.
+
+## Implementing with the VideoGallery
+
+The VideoGallery component is used to display PowerPoint Live sessions. This multi-functional component also supports video streams and screen sharing. Here’s a brief guide to integrating it:
```

Comment:
Help me understand is this a copy/paste error? This docs seems to be related to related to TogetherMode but we have a number of references to PowerPoint live